utils/definitions.py

```diff
@@ -0,0 +1,9 @@
+"""
+Contains some data we use throughout this project.
+Typically this file just contains the project root.
+"""
+from pathlib import Path
+
+
+def get_project_root() -> Path:
+    return Path(__file__).parent.parent
```

Comment:
is this the kind of thing we want to include in our utils across repos?

Reply:
FLASK_ROOT exists in the default config as well currently - we should probably choose one or other. Is there an advantage of having it declared in a separate file to other config?